addons/website_sale_comparison/static/src/js/website_sale_comparison.js

```diff
@@ -26,6 +26,8 @@ var ProductComparison = Widget.extend({
     init: function(){
         this.comparelist_product_ids = JSON.parse(utils.get_cookie('comparelist_product_ids') || '[]');
         this.product_compare_limit = 4;
+        this.guard = null;
+        this.pendingSavingRequests = 0;
```

Comment:
add comment why this is necessary: we want to delay the remove until the last update